src/main/io/vtx_control.h

```diff
@@ -31,6 +31,7 @@ typedef struct vtxChannelActivationCondition_s {
 typedef struct vtxConfig_s {
     vtxChannelActivationCondition_t vtxChannelActivationConditions[MAX_CHANNEL_ACTIVATION_CONDITION_COUNT];
     uint8_t halfDuplex;
+    uint8_t smartAudioEarlyAkkWorkaroundEnable;
```

Comment:
@RomanLut we are missing the default initialization of new setting in the VTX parameter group. This causes initialization to OFF. 
Also, PG version bump is missing

Reply:
1) Do you think default should be ON or OFF?
From one side, OFF will change existing functionality, but from other side, this is workaround, so it should be OFF by default. As I mentioned, new AKK AIO camera does not work with workaround enabled.
2) What do you mean with "PG version bump is missing" ?

Reply:
1. Should be ON by default as compatible with previous versions
2. See here https://github.com/iNavFlight/inav/pull/6477/files#diff-a680ca1e23901579e9214c055509004e8b5c05e3f7281ed74ac1edfc38caabaeR188 as an example. `smartAudioEarlyAkkWorkaroundEnable` should be initialized in `PG_RESET_TEMPLATE` for the VTX config and version should be increased in `PG_REGISTER_WITH_RESET_TEMPLATE`.  In here https://github.com/iNavFlight/inav/blob/master/src/main/io/vtx_control.c#L42

Reply:
Added initialization to ON, increased version.